Client/src/Pages/PageSpeed/CreatePageSpeed/index.jsx

```diff
@@ -333,7 +347,7 @@ const CreatePageSpeed = () => {
             onClick={handleCreateMonitor}
             disabled={Object.keys(errors).length !== 0 && true}
           >
-            Create monitor
+            { isCheckingEndpoint ? "Checking endpoint" : "Create monitor" }
```

Comment:
I think we can skip this, we don't need to change the text of this button.

Instead, you can change it to `LoadingButton` and use the `loading` prop to pass `isLoading` from the reudx state to it.  That will indicate an operation is ongoing.

```
import LoadingButton from "@mui/lab/LoadingButton";
```